workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-category-facet-tabs/ddm-template.ftl

```diff
@@ -8,9 +8,9 @@
 	<ul class="list-unstyled tab-list">
 		<li class="facet-value">
 			<@clay.button
-				cssClass="facet-clear btn-unstyled ${assetCategoriesSearchFacetDisplayContext.isNothingSelected()?then('selected-tab-btn', '')}"
+				cssClass="tab-btn facet-clear btn-unstyled ${assetCategoriesSearchFacetDisplayContext.isNothingSelected()?then('selected-tab-btn', '')}"
```

Comment:
@luisacmn make sure to sort css selectors alphabetically (I also think the formatter will do this for you)